app/js/reducers/supertypes.js

```diff
@@ -45,6 +46,15 @@ const reducer = (state = initialState, action) => {
         };
         return newState;
     }
+    case 'IMPORTABLE_TYPES': {
+        const newState = {
+            ...state,
+            importableTypes: action.importableTypesList.map((item) => {
+                return { value: item, label: item };
+            })
```

Comment:
This could be: `action.importableTypesList.map((item) => ({ value: item, label: item })`, the parenthesis are what lets JS determine that it's returning an object instead of defining a block-scope..